src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -605,6 +622,27 @@ private PrecompileArguments GetPrecompileArgumentsFor(ITaskItem assemblyItem, st
             aotArgs.Add($"static");
         }
 
+        if (UseDirectPInvoke)
```

Comment:
Maybe task should enforce that you don't mix and match UseDirectPInvoke with DirectPInvokes and DirectPInvokeLists.

Reply:
I can add a warning on the task that the `UseDirectPInvoke` will trump `DirectPInvokes` and `DirectPInvokeLists` since that is the current behavior in `aot-compiler.c`, but if we do want to have aot-compiler.c throw an error (instead of the current warning https://github.com/dotnet/runtime/pull/79721#discussion_r1052577384) I can change it to prohibit having the flag and the specific Direct PInvoke parameters.

Reply:
I think we should see it as an error since its an incorrect configuration.

Reply:
Changed it to throw an error now